src/adapters/pg-component.ts

```diff
@@ -0,0 +1,14 @@
+import { createPgComponent, IPgComponent } from '@well-known-components/pg-component'
+import path from 'path'
+
+export async function createDatabase(components: createPgComponent.NeededComponents): Promise<IPgComponent> {
```

Comment:
```suggestion
export async function createDatabaseComponent(components: createPgComponent.NeededComponents): Promise<IPgComponent> {
```

Just in case someone thinks we are creating a database here